docs/CONTRIBUTING.md

```diff
@@ -249,3 +250,48 @@ These contribution guidelines were inspired by and are based on Atom’s
 contribution guidelines. They were modified for the purposes of this repository.
 https://github.com/atom/atom/blob/master/CONTRIBUTING.md - Copyright (c)
 2011-2017 GitHub Inc. (MIT)
+
+### JSDoc Styleguide
```

Comment:
This should come before the paragraph on contribution guidelines (which should maybe be separated by a horizontal rule?)